src/crypto.rs

```diff
@@ -39,6 +42,50 @@ impl PublicKeyExt for PublicKey {
     }
 }
 
+#[cfg(feature = "solana-sdk")]
```

Comment:
Should this feature to be in cargo.toml features list?

Reply:
Actually, it's an "auto-feature" due to `solana-sdk` crate being optional. So here I just check if the dependency is on or off.